docs/apache-airflow/security/api.rst

```diff
@@ -135,6 +135,26 @@ section of ``airflow.cfg``.
 
 Additional options to your auth backend can be configured in ``airflow.cfg``, as a new option.
 
+Enabling CORS
+---------------
+
+[Cross-origin resource sharing (CORS)](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS)
```

Comment:
Please use rst syntax for link.  https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html